image_test.go

```diff
@@ -367,6 +368,55 @@ func TestImageRW(t *testing.T) {
 	}
 }
 
+func TestImageCubeRoundTrip(t *testing.T) {
+	strides := []int{2, 3, 6} // non repretitive, aperiodic strides
+
+	// the data looks like [0, 1, 2, ...]
```

Comment:
the comment doesn't seem to match exactly what is actually in `data`.
I suppose we could just remove it.

Reply:
done (and the comma nitpick above)